cloudflare/validators.go

```diff
@@ -79,3 +81,24 @@ func validateRecordName(t string, value string) error {
 
 	return nil
 }
+
+// validateIntInSlice returns a SchemaValidateFunc which tests if the provided value
+// is of type int
+func validateIntInSlice(valid []int) schema.SchemaValidateFunc {
+	return func(i interface{}, k string) (s []string, es []error) {
```

Comment:
Named result parameters are discouraged, recommend changing to:

```
return func(i interface{}, k string) ([]string, []error) {
  var es []error 
  //...
  // es if invalid
  return nil, es
// nil, nil if valid
  return nil, nil
}
```

This way, `s` is never initialized at all, and both are returned as `nil` if there are no errors.